software/sw_build.mk

```
@@ -63,14 +64,23 @@ TEMPLATE_LDS=src/$@.lds
 
 IOB_SOC_OPENCRYPTOLINUX_CFLAGS ?=-Os -nostdlib -march=rv32imac -mabi=ilp32 --specs=nano.specs -Wcast-align=strict
 
```

We can detect if we are building for simulation with something like this:
```suggestion

ifneq ($(shell grep -s "#define SIMULATION" src/bsp.h),)
SIMULATION=1
endif

```

---

I have not tried to run the system with this modification and the ones below this one.
So it may break the firmware when trying to build for simulation.
Maybe we should keep this part as is, as mentioned by @P-Miranda.